data/openldap_to_389ds/slapd.conf

```diff
@@ -0,0 +1,40 @@
+include		/etc/openldap/schema/core.schema
+include		/etc/openldap/schema/cosine.schema
+include		/etc/openldap/schema/inetorgperson.schema
+include		/etc/openldap/schema/rfc2307bis.schema
+include		/etc/openldap/schema/yast.schema
+
+access to dn.base=""
+        by * read
+
+access to dn.base="cn=Subschema"
+        by * read
+
+access to attrs=userPassword,userPKCS12
+        by self write
+        by * auth
+
+access to attrs=shadowLastChange
+        by self write
+        by * read
+
+access to *
+        by * read
+
+moduleload  back_hdb.la
+moduleload  memberof.la
+
+database	hdb
+suffix		"dc=ldapdom,dc=net"
+checkpoint      1024    5
+cachesize       10000
+rootdn		"cn=root,dc=ldapdom,dc=net"
+rootpw		pass
+directory	/tmp/ldap-sssdtest
+index	objectClass	eq
+
+overlay memberof
+
+#TLSCertificateFile /tmp/ldap-sssdtest.crt
```

Comment:
You didn't test TLS right, is there any bugs or by plan?

Reply:
No TLS currently, by plan

Reply:
Thanks, then I suggest removingt these lines